sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -223,7 +223,7 @@ def _format_url(self, hostname):
         return f"{self.scheme}://{hostname}{self._query_str}"
 
     def _configure_policies(self, **kwargs):
-        credential_policy = _configure_credential(self.credential)
+        credential_policy = _configure_credential(self.credential, self._cosmos_endpoint)
```

Comment:
There should also be the equivalent in `azure.data.tables.aio._base_client_async.py`

Reply:
Resolved in latest commit